modules/troubleshooting-dynamic-plug-in.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plug-ins-reference.adoc
+
+:_content-type: REFERENCE
+[id="troubleshooting-dynamic-plug-in_{context}"]
+= Troubleshooting your dynamic plug-in
+
+Refer to this list of troubleshooting tips if you run into issues loading your plug-in. 
+
+* You have enabled your plug-in in the console Operator configuration. Run the following command to check to check if you have enabled your plug-in is in the console Operator configuration and your plug-in name is the output.
++
+[source,terminal]
+====
+oc get console.operator.openshift.io cluster -o jsonpath='{.spec.plugins}'
+====
+
+** You can also look at the enabled plug-ins on the status card of the *Overview* page in the *Administrator perspective*. You will need to refresh your browser if the plug-in was recently enabled.
+
+* Your plug-in service is healthy
+** Check to see if your plug-in pod status is running and your containers are ready.
+** Check to see if the service label selector matches the pod and the target port is correct.
+** Curl the `plugin-manifest.json` from the service in a terminal on the console pod or another pod on the cluster.
+
+* Your `ConsolePlugin` resource name (`consolePlugin.name`) matches the plug-in name used in `package.json`.
+
+* Your service name, namespace, port, and path are declared correctly in the `ConsolePlugin` resource
+
+* Your plug-in service uses HTTPS and service serving certificates.
+
+* Check for certificates or connection errors in the console pod logs.
+
+* Ensure the feature flag your plug-in relys on is not disabled.
+
+* Your plug-in has `consolePlugin.dependencies` in `package.json` that are not met.
+** This can include console version dependencies or dependencies on other plug-ins. Filter the JS console in your browser for your plug-in's name to see messages that are logged.
+
+* Check to be sure there are no typos in nav extension perspective or section IDs.
+**Your plug-in may be loaded, but nav items missing if IDs are incorrect. Try navigating to a plugin page directly by editing the URL.
+
+* Check see if there are any network policies that are blocking traffic from the console pod to your plug-in service.
+** If necessary, adjust network policies to allow console pods in the openshift-console namespace to make requests to your service.
+
+* You can check the list of dynamic plug-ins to be loaded in your browser in the *Console* tab of the developer tools browser, evaluate `SERVER_FLAGS.consolePlugins` to see the dynamic plug-in on the Console frontend.
```

Comment:
`SERVER_FLAGS.consolePlugins` should be updated to `window.SERVER_FLAGS.consolePlugins`